aws-samples/scaletest-o3de-multiplayer-sample
# Add server log automation

*Description of changes:*

Previously, operators needed to RDP into the server in order to retrieve logs and metrics files generated during testing. This feature syncs test artifacts to a temporary S3 bucket while the server is deployed, and uses an AWS Lambda function to export them to another (external to this tool) S3 bucket when the server stack is destroyed, for longer term storage. 

## Testing

 - successfully deploys
 - manual tests done in Lambda  web console:
    - happy case
    - error case: required bucket not found
    - error case: `resources` event key missing or empty

--
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
